master_xml.py

```diff
@@ -372,7 +374,46 @@ def removeUnreleasedCards(cards):
               "Relict": "Relict",
               "Specter": "Specter",
               "Temeria": "Temeria",
-              "Non_Decoyable": "Stubborn"}
+              "Non_Decoyable": "Stubborn",
+              "Aedirn": "Aedirn",
+              "Alchemy": "Alchemy",
+              "An_Craite": "An Craite",
+              "Bear": "Bear",
+              "Blitz": "Blitz",
+              "Brokvar": "Brokvar",
+              "Cintra": "Cintra",
+              "Devourer": "Devourer",
+              "Dimun": "Dimun",
+              "Double_Agent": "Double Agent",
+              "Draconid": "Draconid",
+              "Dragon": "Dragon",
+              "Drummond": "Drummond",
+              "Harpy": "Harpy",
+              "Heymaey": "Haymaey",
+              "Kaedwen": "Kaedwen",
+              "Leader": "Leader",
+              "Medic": "Medic",
+              "Non_Medicable": "Permadeath",
+              "Officer": "Officer",
+              "Ogroid": "Ogroid",
+              "Organic": "Organic",
+              "Potion": "Potion",
+              "Redania": "Redania",
+              "Regressing": "Regressing",
+              "Relict": "Relict",
+              "Shapeshifter": "Shapeshifter",
+              "Soldier": "Soldier",
+              "Special": "Special",
+              "Spell": "Spell",
+              "Spy": "Agent",
+              "Support": "Support",
+              "Svalblod": "Svalblod",
+              "Tactic": "Tactic",
+              "Temeria": "Temeria",
+              "Tordarroch": "Tordarroch",
+              "Tuirseach": "Tuirseach",
+              "Vodyanoi": "Vodyanoi",
+              "Weather": "Weather"}
```

Comment:
dupe... just put this through a JSON validator

Reply:
Fixed. Sorry was in a rush last night when I typed these up. I've removed dupes and alphabetised them.